backupstoragelocation.md

```diff
@@ -74,12 +74,13 @@ spec:
     kmsKeyId: "502b409c-4da1-419f-a16e-eif453b3i49f"
     
     # Specify the file that contains the SSE-C customer key to enable customer key encryption of the backups
-    # stored in S3. The referenced file should contain a 32-byte string.
+    # stored in S3. The referenced file should exist within the velero container and
+    # should contain a 32-byte string. It is typically mounted from a secret.
     #  
-    # The customerKeyEncryptionFile points to a mounted secret within the velero container.
-    # Add the below values to the velero cloud-credentials secret:
+    # Eg. add to the velero "cloud-credentials" secret this entry with the base64 encoded key
+    # (will be decoded when the secret is mounted)
     # customer-key: <your_b64_encoded_32byte_string>
```

Comment:
Should this line be uncommented? or is the encoded customer-key only supposed to be populated by the object_store.go?

Reply:
Can it be expected that someone put in encoded 32 byte string instead of specifying the file name in the key below this one?
If so
```suggestion
    customer-key: <your_b64_encoded_32byte_string>
```

Reply:
The function `readCustomerKey` currently only supports reading the secret from a file mounted inside the velero container. So the example here shows how to put the key into the secret, therefore it is just a comment and not part of the BSL configuration. I think it make sense, _not_ to provide the senstive key as parameter, but only as secret.